_posts/2020-04-20-schedule-your-functions.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "On time - schedule your OpenFaaS functions"
```

Comment:
I like the "on time" pun, what if you put it into the intro rather than the description?

Something like this might be more descriptive and better for SEO:

"Keep your functions running to schedule with the cron-connector."

Reply:
Yeah ok :+1: Done :heavy_check_mark: